tools/training-aev-benchmark.py

```diff
@@ -85,7 +99,20 @@ def print_timer(label, t):
     print(f'{label} - {t}')
 
 
-def benchmark(parser, dataset, use_cuda_extension, force_inference=False):
+def format_time(t):
+    if t < 1:
+        t = f'{t * 1000:.1f} ms'
+    else:
+        t = f'{t:.3f} sec'
+    return t
+
+
+def benchmark(parser, dataset, use_cuda_extension, force_train=False):
+    global summary
+    global runcounter
+
```

Comment:
Why do you need these two lines?

Reply:
they will be modified at the end of this function